exporter/logzioexporter/logziospan.go

```diff
@@ -16,22 +15,107 @@ const (
 	tagDotReplacementCharacter = "@"
 )
 
-// logzioSpan is same as esSpan with a few different json field names and an addition on type field.
```

Comment:
I strongly advise to structure your PR such that it does not move code unnecessarily. The diff view should be showing no changes to this type, instead for me it shows as deleted and another one added below, which is named differently and may or may not be the same. Why introduce this additional complexity for the reviewer?

Reply:
@zzzk1 I agree with @yurishkuro